src/utils/suggestions.js

```diff
@@ -21,67 +22,1300 @@ const extractSuggestion = (val) => {
 };
 
 function replaceDiacritics(s) {
-	let str = s ? String(s) : '';
-
-	const diacritics = [
-		/[\300-\306]/g,
-		/[\340-\346]/g, // A, a
-		/[\310-\313]/g,
-		/[\350-\353]/g, // E, e
-		/[\314-\317]/g,
-		/[\354-\357]/g, // I, i
-		/[\322-\330]/g,
-		/[\362-\370]/g, // O, o
-		/[\331-\334]/g,
-		/[\371-\374]/g, // U, u
-		/[\321]/g,
-		/[\361]/g, // N, n
-		/[\307]/g,
-		/[\347]/g, // C, c
-	];
-
-	const chars = ['A', 'a', 'E', 'e', 'I', 'i', 'O', 'o', 'U', 'u', 'N', 'n', 'C', 'c'];
-
-	for (let i = 0; i < diacritics.length; i += 1) {
-		str = str.replace(diacritics[i], chars[i]);
+	let str = s ? String(s) : "";
+
+	const diacritics = {
```

Comment:
@lakhansamani This file is getting larger because of this constant. Please move it to a different file.